plugins/newsdownloader.koplugin/main.lua

```diff
@@ -613,7 +611,7 @@ function NewsDownloader:processFeed(feed_type, feeds, cookies, limit, download_f
             self:createFromDescription(
                 feed,
                 feed_title,
-                feed_description,
+                feed_description or "No description available",
```

Comment:
I'd rather keep the logic up above for clarity. (And with a plain empty string.)